vm/src/stdlib/os.rs

```diff
@@ -2080,27 +2080,27 @@ mod posix {
 
     #[pyfunction(name = "WIFSIGNALED")]
     fn wifsignaled(status: i32) -> bool {
-        unsafe { libc::WIFSIGNALED(status) }
+        libc::WIFSIGNALED(status)
```

Comment:
what's happening here? is this safe in linux but unsafe in macos?

Reply:
Looks like it, I guess I'll just put `#[allow(unused_unsafe)]`

Reply:
https://github.com/rust-lang/libc/issues/1888

Reply:
how about fixing libc as 0.2.76 for a while? It seems they will be all safe after fixing the issue